<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-06-25T11:16:49Z" name="Surefire suite" finished-at="2019-06-25T11:17:13Z" duration-ms="23754">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(cucumber.api.testng.PickleEventWrapper, cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:runners.RunCukesTest@78aab498]" name="runScenario" class="cucumber.api.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2019-06-25T11:16:49Z" name="Surefire test" finished-at="2019-06-25T11:17:13Z" duration-ms="23754">
      <class name="runners.RunCukesTest">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:runners.RunCukesTest@78aab498]" started-at="2019-06-25T11:16:49Z" name="setUpClass" finished-at="2019-06-25T11:16:53Z" duration-ms="3642" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(cucumber.api.testng.PickleEventWrapper, cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:runners.RunCukesTest@78aab498]" started-at="2019-06-25T11:16:53Z" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2019-06-25T11:17:11Z" duration-ms="17920" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Register with valid email"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Registration"]]>
              </value>
            </param>
          </params>
          <exception class="cucumber.runtime.CucumberException">
            <message>
              <![CDATA[Could not convert arguments for step [^I enter all the mandatory information$] defined at 'project.stepdefs.RegisterSteps.I_Enter_My_Regular_Income_Sources(Transaction) in file:/C:/Users/gayat/eclipse-workspace/belong-test-exercise/target/classes/'.
The details are in the stacktrace below.]]>
            </message>
            <full-stacktrace>
              <![CDATA[cucumber.runtime.CucumberException: Could not convert arguments for step [^I enter all the mandatory information$] defined at 'project.stepdefs.RegisterSteps.I_Enter_My_Regular_Income_Sources(Transaction) in file:/C:/Users/gayat/eclipse-workspace/belong-test-exercise/target/classes/'.
The details are in the stacktrace below.
at cucumber.runner.PickleStepDefinitionMatch.couldNotConvertArguments(PickleStepDefinitionMatch.java:69)
at cucumber.runner.PickleStepDefinitionMatch.runStep(PickleStepDefinitionMatch.java:46)
at cucumber.runner.TestStep.executeStep(TestStep.java:63)
at cucumber.runner.TestStep.run(TestStep.java:49)
at cucumber.runner.PickleStepTestStep.run(PickleStepTestStep.java:43)
at cucumber.runner.TestCase.run(TestCase.java:44)
at cucumber.runner.Runner.runPickle(Runner.java:40)
at cucumber.api.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:68)
at cucumber.api.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: io.cucumber.datatable.CucumberDataTableException: 'project.dataobjects.Transaction' could not transform
      | First name | Last name | Email                       | Password | First name | Last name | Address  | City  | State    | Zip/Postal Code | Country       | Mobile phone |
      | G          | T         | gayathri.tiwari01@gmail.com | Gaya3    | G          | T         | Mumba St | Texas | Victoria | 30000           | United States | 04455667788  |

at io.cucumber.datatable.DataTableType.transform(DataTableType.java:162)
at io.cucumber.datatable.DataTableTypeRegistryTableConverter.convert(DataTableTypeRegistryTableConverter.java:53)
at io.cucumber.datatable.DataTable.convert(DataTable.java:362)
at io.cucumber.stepexpression.StepExpressionFactory$3.transform(StepExpressionFactory.java:73)
at io.cucumber.stepexpression.DataTableArgument.getValue(DataTableArgument.java:19)
at cucumber.runner.PickleStepDefinitionMatch.runStep(PickleStepDefinitionMatch.java:41)
... 37 more
Caused by: io.cucumber.datatable.CucumberDataTableException: Can't convert DataTable to Map<java.lang.String, java.lang.String>. Encountered duplicate key First name with values G and G
at io.cucumber.datatable.CucumberDataTableException.cantConvertToMap(CucumberDataTableException.java:25)
at io.cucumber.datatable.CucumberDataTableException.duplicateKeyException(CucumberDataTableException.java:30)
at io.cucumber.datatable.DataTable.asMaps(DataTable.java:254)
at project.datatables.Configurer$2.transform(Configurer.java:42)
at project.datatables.Configurer$2.transform(Configurer.java:39)
at io.cucumber.datatable.DataTableType$TableTransformerAdaptor.transform(DataTableType.java:294)
at io.cucumber.datatable.DataTableType.transform(DataTableType.java:159)
... 42 more
]]>
            </full-stacktrace>
          </exception> <!-- cucumber.runtime.CucumberException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:runners.RunCukesTest@78aab498]" started-at="2019-06-25T11:17:11Z" name="tearDownClass" finished-at="2019-06-25T11:17:13Z" duration-ms="2040" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- runners.RunCukesTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
